addons/test_mail/tests/test_message_post.py

```diff
@@ -280,6 +280,25 @@ def test_post_log(self):
         self.assertEqual(new_note.email_from, formataddr((self.user_employee.name, self.user_employee.email)))
         self.assertEqual(new_note.notified_partner_ids, self.env['res.partner'])
 
+    def test_post_log_with_view(self):
+        channel = self.env['mail.channel'].create({
+            'name': 'Test Channel',
+            'moderation_guidelines_msg': 'Test message'
+        })
+        values = {
+            'partner': self.user_employee.partner_id,
+            'channel': channel
+        }
+        new_note = self.test_record.with_user(self.user_employee)._message_log_with_view(
+            'mail.mail_channel_send_guidelines', values=values,
```

Comment:
If possible do not use existing data. I think you may add a small template running on mail.test.simple as data in test_mail/data, as any other template. We could use it in other tests afterwards